ui/lib/apps/ClusterInfo/pages/List.tsx

```diff
@@ -56,14 +56,20 @@ export default function ListPage() {
     },
   ]
 
+  function clickTab(tabKey) {
+    if (tabKey === 'instance') {
```

Comment:
This is too hacky, how about using a redirection (redirect `/ `to `/instance`)?

Reply:
Yep, I do think so. It is redirection before.

```
<Route	       
            path="/cluster_info"
            element={<Navigate to="/cluster_info/instance" replace />}
```

I do this change because I don't want the page change event to happen when we are on the "/cluster_info/instance" page and clicking the "Cluster Info" link on the left side menu, to keep the same behavior as other apps.

Let me think it about more to find a more elegant solution.

Reply:
I think this behaviour is fine, since this page change event only generates a dummy "/cluster_info" telemetry information, which we can just ignored, considering that a `/cluster_info/instance` should be always followed and information is not lost?

Reply:
Ok, I reverted this part code.